sda-commons-server-s3/build.gradle

```diff
@@ -3,10 +3,10 @@ dependencies {
   api project(':sda-commons-server-healthcheck')
   api 'com.amazonaws:aws-java-sdk-s3'
   api 'org.slf4j:jcl-over-slf4j'
-  api 'io.opentracing.contrib:opentracing-aws-sdk-1'
-  api 'io.opentracing:opentracing-util'
+
+  implementation 'io.opentelemetry.instrumentation:opentelemetry-aws-sdk-1.11'
```

Comment:
Extract version to dependencies module?

Reply:
Quite tricky but that is the library name `opentelemetry-aws-sdk-1.11`.
They have the convention to have the min supported version in the module name so that they are able to support multiple versions of a library